endpoints/openrtb2/auction_test.go

```diff
@@ -3491,3 +3491,32 @@ type hardcodedResponseIPValidator struct {
 func (v hardcodedResponseIPValidator) IsValid(net.IP, iputil.IPVersion) bool {
 	return v.response
 }
+
+func TestValidateBanner(t *testing.T) {
+	impIndex := 4
```

Comment:
Looks like the impIndex is just used for the error messages in `validateBanner`. Why not use 0?

Reply:
no reason. I can change to 0